### PR TITLE
Initial implementation of the Markdown Linter

### PR DESCRIPTION
**Pull request by aia**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -84,6 +84,16 @@ jobs:
...
+  markdown-link-check:
+    name: Markdown linting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Check out the code
+      - name: Lint Code Base
+        uses: docker://avtodev/markdown-lint:v1
+        with:
+          args: "./gamestonk_terminal/options/README.md"
```
Comment (DidierRLopes) [inline .github/workflows/test.yml:96]: Do you want to check options README?

This file should be changed drastically to only have a high-level detail of what Options contains. Similar to this: https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/gamestonk_terminal/stocks

Comment (aia) [reply .github/workflows/test.yml:96]: In a separate PR, yes.

Comment (DidierRLopes) [reply .github/workflows/test.yml:96]: Why on the workflow do we not use as args "." the entire codebase?

Comment (aia) [reply .github/workflows/test.yml:96]: > Why on the workflow do we not use as args "." the entire codebase?

We have too many linting errors in our existing markdowns. I want to add and fix them one at a time.

Comment (DidierRLopes) [reply .github/workflows/test.yml:96]: Sounds good. Probably we can wait for the markdown refactoring that will happen soon for this.

**Review by DidierRLopes — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (aia) [reply .github/workflows/test.yml:96]: I keep looking at all the red that I have in VScode every time I open a markdown from the repo and I decided that this is the time to get it fixed :-)

**Pull request merged**